From 4f8f5ea9623f8e85fa0c0c603af62b4048672429 Mon Sep 17 00:00:00 2001
From: Gaosheng Cui <cuigaosheng1@huawei.com>
Date: Wed, 28 Jun 2023 23:45:20 -0400
Subject: [PATCH 238/780] [Huawei] Revert "[Huawei] cgroup: Remove the
 unnecessary braces in css_task_iter_stop"

Offering: HULK
hulk inclusion
category: others
bugzilla: 188807

--------------------------------

This reverts commit e734f8e02494d25b6a68b0e53dc6d4ef2482455e.

Signed-off-by: Gaosheng Cui <cuigaosheng1@huawei.com>
---
 kernel/cgroup/cgroup.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/kernel/cgroup/cgroup.c b/kernel/cgroup/cgroup.c
index 1574312b307d..b725420ee06a 100644
--- a/kernel/cgroup/cgroup.c
+++ b/kernel/cgroup/cgroup.c
@@ -4758,8 +4758,9 @@ static void css_task_iter_stop(struct css_task_iter *it,
 {
 	lockdep_assert_held(&css_set_lock);
 
-	if (it->ss == ss)
+	if (it->ss == ss) {
 		it->flags |= CSS_TASK_ITER_STOPPED;
+	}
 }
 
 static void css_task_iter_advance(struct css_task_iter *it)
-- 
2.17.1

